LncRNAanalysisPipe.nf

```diff
@@ -69,11 +58,11 @@ def print_white = {  str -> ANSI_WHITE + str + ANSI_RESET }
 
 //Help information
 // Nextflow  version
-version="v0.2.42"
+version="v0.2.5"
 //=======================================================================================
 // Nextflow Version check
-if( !nextflow.version.matches('0.26+') ) {

```

Comment:
No need for that, you can use the manifest version information to check for this kind of stuff and remove the boilerplate here...